### DISCUSSION TITLE
Does decontx support 10x HT samples? How to adjust delta for HT samples?

### DISCUSSION THREAD

**Discussion by MeshifLu** [campbio/celda#394] [General]
Hi celda team,

I am currently trying to apply DecontX on our 10x HT data and encounter some problems. I have tried DecontX on our 10x standard samples and they all look good. However, running the pipeline with default parameters on HT samples detects lots of ambient RNAs in multiple samples. 
![decontx_27_](https://github.com/campbio/celda/assets/20402765/f7918546-d3c2-4ff1-8862-b32f3df0db58)

So, my questions are:
1. Have you tested 10x HT data? Is it normal to see high contamination level in HT?

2. Since the contamination is high in several HT samples, I have also tried to adjust delta parameter based on the default delta value. 
```
deltaV <- metadata(sce.decontX)$decontX$estimates$all_cells$delta
for (deltaPct in c(1.0, 0.9, 0.8)) {
  sce.decontX.delta <- decontX(sce.decontX, delta = c(deltaV[1], deltaV[2]*deltaPct), estimateDelta = FALSE)
  plot(sce.decontX$decontX_contamination, sce.decontX.delta$decontX_contamination,
          main = paste0("deltaPct: ",deltaPct ),
          xlab = "DecontX estimated priors", ylab = "Setting priors to estimate lower contamination")
  abline(0, 1, col = "red", lwd = 2)
}
```
I suppose that the result should be identical to the default one when deltaPct=1.0. However, I got very distinct patterns when deltaPct equal to 1.0. Did I do something wrong? What's the better way to remove contamination on HT samples?
![decontx_27_estimated_priors](https://github.com/campbio/celda/assets/20402765/42c5a73e-be6b-4162-a214-40385481bddd)

**Comment by joshua-d-campbell** [comment]
Hello,
Thanks for using our tool. We haven't thoroughly tested it on HT data yet. The bigger factor may be the cluster labels. DecontX assumes each cell is a mixture of its true cell population (defined by its cluster) and ambient RNA from all other clusters. If the group of cells that is all high contamination in the middle of the UMAP is in a cluster with other cell types, then it may get inappropriately flagged. You can try running another clustering workflow like Seurat and inputting their cluster labels into the `z` parameter. Can you show the plot of cluster labels estimated by decontX?

**Reply by MeshifLu** [reply]
Thank you for your explanation. Our samples contain lots of neuronal cells which are very similar with each other. They tend to form a big cluster in the center of the UMAP and most of contamination signals are from this cluster.
![decontx_27_wCluster](https://github.com/campbio/celda/assets/20402765/05d59a45-f7fd-4216-be68-7a07010b10bf)

I will try to use different clustering methods and change cluster labels to see whether this can be improved.
